sql/core/src/test/resources/sql-tests/results/columnresolution-negative.sql.out

```diff
@@ -161,7 +161,7 @@ SELECT db1.t1.i1 FROM t1, mydb2.t1
 struct<>
 -- !query 18 output
 org.apache.spark.sql.AnalysisException
-cannot resolve '`db1.t1.i1`' given input columns: [mydb2.t1.i1, mydb2.t1.i1]; line 1 pos 7
+cannot resolve '`db1`.`t1`.`i1`' given input columns: [mydb2.t1.i1, mydb2.t1.i1]; line 1 pos 7
```

Comment:
do you think we should just make `sql` same as `name`? It looks to me that `'db1.t1.i1'` is better than `` '`db1`.`t1`.`i1`' ``, as it's more compact and is not ambiguous.

Reply:
Yes. I agree. For the four examples, we will have the following results:
```
$"a.b".expr.asInstanceOf[org.apache.spark.sql.catalyst.analysis.UnresolvedAttribute].sql
```
with my previous fix: 
```
`a`.`b` 
```     
make sql same as name: 
```
a.b
```
```
$"`a.b`".expr.asInstanceOf[org.apache.spark.sql.catalyst.analysis.UnresolvedAttribute].sql
```
with my previous fix: 
```
`a`.`b`
```      
make sql same as name: 
```
`a.b`
```
```
$"`a`.b".expr.asInstanceOf[org.apache.spark.sql.catalyst.analysis.UnresolvedAttribute].sql
```
with my previous fix: 
```
`a`.`b`     
```
make sql same as name: 
```
a.b
```
```
$"`a.b`.c".expr.asInstanceOf[org.apache.spark.sql.catalyst.analysis.UnresolvedAttribute].sql
```
with my previous fix: 
```
`a.b`.`c`
```     
make sql same as name: 
```
`a.b`.c         
```
Does this look good to everybody?

Reply:
LGTM

Reply:
@huaxingao Were you planning to make a change to make `name` same as `sql` ? Currently they are different ?

Reply:
@cloud-fan I had a question, the 3rd example from huaxin, wanted to confirm again if the output looks okay to you.

**from huaxin**
```
$"`a`.b".expr.asInstanceOf[org.apache.spark.sql.catalyst.analysis.UnresolvedAttribute].sql
```
with my previous fix:
```
`a`.`b`
```     
make sql same as name:
```
a.b
```
Is it okay to drop the backtick from the first identifier ?

Reply:
@dilipbiswal 
Currently ```sql``` is not the same as ```name```

Reply:
@cloud-fan Makes sense. Thanks for clarification...